spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 require "yuriita"
 
 require "support/tokens"
+require "support/matchers/produce_tokens"
```

Comment:
Did you get a failure when attempting to run the lexer_spec by itself?

Reply:
Yes, without this require I get a `NoMethodError` for every example in `lexer_spec.rb`:

```
      NoMethodError:
        undefined method `produce_tokens' for #<RSpec::ExampleGroups::YuriitaLexer::Lex:0x00007f91c70a9000>
      # ./spec/yuriita/lexer_spec.rb:105:in `block (3 levels) in <top (required)>'
```

Reply:
👍 I think I ran into this with `tokens` but must have somehow missed the `produce_tokens` matcher.